sql/core/pom.xml

```diff
@@ -113,6 +113,10 @@
       <groupId>com.fasterxml.jackson.core</groupId>
       <artifactId>jackson-databind</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.codehaus.jackson</groupId>
+      <artifactId>jackson-mapper-asl</artifactId>
```

Comment:
In hive module, it was inevitable due to Apache Hive, but this will affect Spark's `mllib` module and `kafka-0-10-sql` module, too.